addons/l10n_in_ewaybill/models/l10n_in_ewaybill.py

```diff
@@ -704,6 +714,27 @@ def _set_data_from_attachment(self):
                 "ewaybill_expiry_date": ewb_validity,
             })
 
+    def _generate_and_attach_pdf(self, report_name, doc_label):
```

Comment:
```suggestion
    def _generate_and_attach_pdf(self, doc_label):
```
I don't think we need an extra param for report name